lib/react_test_utils.dart

```diff
@@ -0,0 +1,255 @@
+/// and returns that one result, or throws exception if there is any other
+/// number of matches besides one.
+JsObject findRenderedDOMComponentWithTag(JsObject tree, String tag) {
+  return _TestUtils.callMethod(
+      'findRenderedDOMComponentWithTag', [tree, tag]);
+}
+
+/// Same as scryRenderedComponentsWithType() but expects there to be one result
+/// and returns that one result, or throws exception if there is any other
+/// number of matches besides one.
+JsObject findRenderedComponentWithType(
+    JsObject tree, ReactComponentFactory componentType) {
+  return _TestUtils.callMethod(
+      'findRenderedComponentWithType', [tree, getFactory(componentType)]);
+}
+
+/// Returns true if element is a composite component.
+/// (created with React.createClass()).
+bool isCompositeComponent(JsObject instance) {
+  return _TestUtils.callMethod('isCompositeComponent', [instance]);
+}
+
+/// Returns true if instance is a composite component.
+/// (created with React.createClass()) whose type is of a React componentClass.
+bool isCompositeComponentWithType(
+    JsObject instance, ReactComponentFactory componentClass) {
+  return _TestUtils.callMethod(
+      'isCompositeComponentWithType', [instance, getFactory(componentClass)]);
+}
+
+/// Returns true if instance is a DOM component (such as a <div> or <span>).
+bool isDOMComponent(JsObject instance) {
+  return _TestUtils.callMethod('isDOMComponent', [instance]);
+}
+
+/// Returns true if element is any ReactElement.
+bool isElement(element) {
+  return _TestUtils.callMethod('isElement', [element]);
+}
+
+/// Returns true if element is a ReactElement whose type is of a
+/// React componentClass.
+bool isElementOfType(JsObject element, ReactComponentFactory componentClass) {
+  return _TestUtils.callMethod(
+      'isElementOfType', [element, getFactory(componentClass)]);
+}
+
+/// Finds all instances of components with type equal to componentClass.
+JsObject scryRenderedComponentsWithType(
+    JsObject tree, ReactComponentFactory componentClass) {
+  return _TestUtils.callMethod(
+      'scryRenderedComponentsWithType', [tree, getFactory(componentClass)]);
+}
+
+/// Finds all instances of components in the rendered tree that are DOM
+/// components with the class name matching className.
+List scryRenderedDOMComponentsWithClass(JsObject tree, String className) {
+  return _TestUtils.callMethod(
+      'scryRenderedDOMComponentsWithClass', [tree, className]);
+}
+
+/// Finds all instances of components in the rendered tree that are DOM
+/// components with the tag name matching tagName.
+JsObject scryRenderedDOMComponentsWithTag(JsObject tree, String tagName) {
+  return _TestUtils.callMethod(
+      'scryRenderedDOMComponentsWithTag', [tree, tagName]);
+}
+
+/// Render a Component into a detached DOM node in the document.
+JsObject renderIntoDocument(JsObject instance) {
+  var div = new DivElement();
+  return _TestUtils.callMethod('renderIntoDocument', [instance, div]);
```

Comment:
There's actually no second argument to [this method](https://github.com/matthewbalvanz-wf/react-dart/blob/76bbed05ed4bcb3acc3dd4203e49ac6aceb0aa66/lib/react_with_addons.js#L14116), so the `div` variable can be eliminated.